### ISSUE TITLE
Security report

### ISSUE THREAD

**Issue by jakubboucek** [hejny/loader#3]
This library has mass of security issues thats allows more attack vector.

It this library an serious product, or any unimportant output, like a prototype for example? Are you interest of more details @hejny?

**Comment by jakubboucek** [comment]
ping! it's interesting for4 or not?
if not pls close issue

**Comment by hejny** [comment]
No, unfortunately, I have no time to fix it :( I am going to make this repo private (and keep your access).
